src/SigSpec.Core/SigSpecGeneratorSettings.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using Newtonsoft.Json;
-using NJsonSchema.Generation;
+using NJsonSchema.NewtonsoftJson.Generation;
 
 namespace SigSpec.Core
 {
-    public class SigSpecGeneratorSettings : JsonSchemaGeneratorSettings
+    public class SigSpecGeneratorSettings : NewtonsoftJsonSchemaGeneratorSettings
```

Comment:
I assume the default was newtonsoft before?

Reply:
It was Newtonsoft before, he just extracted all Newtonsoft out into a separate package with better names

Reply:
For example on bus:

![image](https://github.com/torutek/SigSpec/assets/47214243/01cb3df0-499c-4776-b177-fc1ba65cb7f5)